From 1dcb1532b308b69370f16b7c644e0c42808d5af8 Mon Sep 17 00:00:00 2001
From: Tim Schumacher <timschumi@gmx.de>
Date: Mon, 15 Apr 2019 10:40:10 +0200
Subject: [PATCH] Bump Security Patch Level to 2019-03-01

CVE-2018-20346	A-121156452	High
CVE-2019-1985	A-118694079*	High
CVE-2019-2003	A-116321860	High	Change 243758
CVE-2019-2004	A-115739809	High
CVE-2019-1989	A-118399205	Crit.	No libavc
CVE-2019-1990	A-118453553	Crit.	No libhevc
CVE-2019-2009	A-120665616	Crit.	No l2c_lcc_proc_pdu
CVE-2019-2010	A-118152591	High	Change 244134
CVE-2019-2012	A-120497437	High	Change 244138
CVE-2019-2013	A-120497583	High	Change 244138
CVE-2019-2014	A-120499324	High	Change 244139
CVE-2019-2015	A-120503926	High	Change 244141
CVE-2019-2016	A-120664978	High	Change 244142
CVE-2019-2017	A-121035711	High	Change 244147
CVE-2018-9561	A-111660010[1]	High	Change 244161
CVE-2018-9561	A-111660010[2]	High	Change 244162
CVE-2018-9563	A-114237888[1]	High	Change 244161
CVE-2018-9563	A-114237888[2]	High	Change 244162
CVE-2018-9564	A-114238578[1]	High	Change 244161
CVE-2018-9564	A-114238578[2]	High	Change 244162
CVE-2019-2019	A-115635871	High	Change 244242
CVE-2019-2020	A-116788646	High	Change 244243
CVE-2019-2021	A-120428041	High	Change 244245
CVE-2019-2022	A-120506143	High	Change 244138

Reference version: 7.0

Change-Id: I0fa56f3bb347a885a4ac7d09d2f2da7eb71af1e5
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index fb7beec..53fa328 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -81,7 +81,7 @@ ifeq "" "$(PLATFORM_SECURITY_PATCH)"
   # Can be an arbitrary string, but must be a single word.
   #
   # If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-  PLATFORM_SECURITY_PATCH := 2019-02-01
+  PLATFORM_SECURITY_PATCH := 2019-03-01
 endif
 
 ifeq "" "$(BUILD_ID)"
-- 
2.7.4

